tests/e2e/cht-form/default/delivery.wdio-spec.js

```diff
@@ -0,0 +1,129 @@
+    await genericForm.nextPage();
+
+    const summaryInfo = await deliveryForm.getSummaryInfo();
+    expect(summaryInfo.womanCondition).to.equal('Alive and well');
+    expect(summaryInfo.deliveryDate).to.equal(DATE);
+    expect(summaryInfo.deliveryPlace).to.equal('Health facility');
+    expect(summaryInfo.deliveredBabies).to.equal('1');
+    expect(summaryInfo.deceasedBabies).to.equal('0');
+    expect(summaryInfo.pncVisits).to.equal('None');
+
+    const data = await mockConfig.submitForm();
+    const jsonObjMother = data[0].fields;
+    const jsonObjBaby = data[1];
+
+    expect(jsonObjMother.patient_uuid).to.equal('12345');
+    expect(jsonObjMother.patient_id).to.equal('79376');
+    expect(jsonObjMother.patient_name).to.equal('Pregnant Woman');
+    expect(jsonObjMother.data.meta.__pregnancy_uuid).to.equal('test UUID');
+    expect(jsonObjMother.condition.woman_outcome).to.equal('alive_well');
+    expect(jsonObjMother.pnc_danger_sign_check.r_pnc_danger_sign_present).to.equal('no');
+    expect(jsonObjMother.delivery_outcome.babies_delivered).to.equal('1');
+    expect(jsonObjMother.delivery_outcome.babies_alive).to.equal('1');
+    expect(jsonObjMother.delivery_outcome.delivery_date).to.equal(DATE);
+    expect(jsonObjMother.delivery_outcome.delivery_place).to.equal('health_facility');
+    expect(jsonObjMother.delivery_outcome.delivery_mode).to.equal('vaginal');
+    expect(jsonObjMother.babys_condition.baby_repeat[0].baby_details.baby_condition).to.equal('alive_well');
+    expect(jsonObjMother.babys_condition.baby_repeat[0].baby_details.baby_name).to.equal(BABY_NAME);
+    expect(jsonObjMother.babys_condition.baby_repeat[0].baby_details.baby_sex).to.equal(BABY_SEX);
+    expect(jsonObjMother.babys_condition.r_baby_danger_sign_present_any).to.equal('no');
+
+    expect(jsonObjBaby.name).to.equal(BABY_NAME);
+    expect(jsonObjBaby.sex).to.equal(BABY_SEX);
+    expect(jsonObjBaby.date_of_birth).to.equal(DATE);
+    expect(jsonObjBaby.t_danger_signs_referral_follow_up).to.equal('no');
+  });
+
+  it('should submit a delivery - Death mother and baby', async () => {
+    await mockConfig.startMockApp('default', 'app', 'delivery');
+
+    await browser.execute(() => {
+      const myForm = document.getElementById('myform');
+      myForm.contactSummary = { pregnancy_uuid: 'test UUID' };
+      myForm.content = { contact: { _id: '12345' } };
+      myForm.user = { phone: '+50689999999' };
+    });
+
+    const title  = await genericForm.getFormTitle();
+    expect(title).to.equal('Delivery');
+
+    await deliveryForm.selectDeliveryConditionWomanOutcome('deceased');
+    await genericForm.nextPage();
+    await deliveryForm.fillWomanDeathInformation({
+      date: DATE,
+      place: 'health_facility',
+      notes: 'Test notes - Mother\'s death'
+    });
+    await genericForm.nextPage();
+    await genericForm.nextPage();
+
+    const data = await mockConfig.submitForm();
+    const jsonObj = data[1];
```

Comment:
Can you also assert the contents of `data[0]` (the actual delivery form)?  With these forms that are resulting in multiple docs, I think we want to be extra clear about what we expect to be in each of these docs.

Reply:
Of course. Done 😄